.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: php
 
-sudo: false
```

Comment:
Hi @stof, the original configuration is false.
Please see this [PR](https://github.com/composer/spdx-licenses/pull/10). Thanks.